src/helpers/filterRemoteGroups.js

```diff
@@ -0,0 +1,6 @@
+const uniqBy = require('lodash/uniqBy')
+
+const filterRemoteGroups = groups => {
+  return uniqBy(groups, 'gid').filter(({ structure, gid }) => gid && structure)
+}
```

Comment:
I think the name filterRemoteGroups could be better, I did not understand what it meant when reading it. 

`filterValidGroup` ?